thirdrough.css

```diff
@@ -0,0 +1,293 @@
+.{
+	margin: 0;
+	padding: 0;
+}
+body{
+		}
+div.section-top a{

```

Comment:
follow styling guide like 2 spaces instead of 4 spaces

instead of 

```css
a:hover {
    color: white
}

use

a:hover {
  color: white
}
```
Note the spacing, and space before open curly brace